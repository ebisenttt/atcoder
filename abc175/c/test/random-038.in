-995440403131261 840062074719922 633431392065206
