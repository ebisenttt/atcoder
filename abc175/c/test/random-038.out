271422380999151
